compiler/rustc_data_structures/src/marker.rs

```diff
@@ -147,14 +147,13 @@ cfg_match! {
             [crate::owned_slice::OwnedSlice]
         );
 
-        // MIPS, PowerPC and SPARC platforms with 32-bit pointers do not
-        // have AtomicU64 type.
-        #[cfg(not(any(target_arch = "powerpc", target_arch = "powerpc", target_arch = "sparc")))]
+        // Use portable AtomicU64 for targets without native 64-bit atomics
```

Comment:
Wrong comment position, should be placed before line 156.

Reply:
The comment applies to both the following statements, see the comment from the previous version.